src/transformers/models/llava/processing_llava.py

```diff
@@ -70,12 +79,14 @@ def __init__(
         image_processor=None,
         tokenizer=None,
         patch_size=None,
+        num_additional_image_tokens=0,
```

Comment:
should probably be placed at the end for BC as well 😉

Reply:
also it was set to `1` (i mean `num_image_tokens -= 1` now it will use 0

Reply:
this one is important

Reply:
oke, will fix it and merge after making sure tests pass